spec/periodicChecksSpec.js

```diff
@@ -856,5 +856,54 @@ describe('Periodic Checks Module', () => {
         });
       });
     });
+
+    describe('Should Check If a Opened Issue is Assigned to a project', () => {
+      beforeEach(async () => {
+        github.issues.listForRepo = jasmine
+          .createSpy('listForRepo')
+          .and.resolveTo({
+            data: [
+              issues.withProject,
+              issues.anotherWithProject,
+              issues.withoutProject,
+            ],
+          });
+        spyOn(periodicCheckModule, 'ensureNewIssuesHaveProjects').
+          and.callThrough();
+        await robot.receive(payloadData);
```

Comment:
The payload for this event should be `issued.opened` since this feature works when an issue is opened. The default `payloadData` event and action si `schedule.repository`. https://github.com/oppia/oppiabot/blob/a9c06dc340450a5d40cd3aa42bebaec421650b7c/fixtures/periodicCheckPayload.json#L1-L4

Reply:
Sure Sir, Changing it right now. Please let me know if any other change is required. I will be happy to fix. Thanks.

Reply:
Sorry I had a typo in my earlier message. It should be `issues.opened` not `issued`.

Reply:
Changing it right away Sir. Thanks.